convert.c

```diff
@@ -256,6 +257,117 @@ static int will_convert_lf_to_crlf(size_t len, struct text_stat *stats,
 
 }
 
+static struct encoding {
+	const char *name;
+	iconv_t *to_git;
+	iconv_t *to_worktree;
```

Comment:
Maybe this could briefly described by a comment like this:

> iconv handle for conversion back to the user-defined character encoding

Reply:
ok!